kyuubi-server/src/main/scala/org/apache/kyuubi/credentials/HadoopFsDelegationTokenProvider.scala

```diff
@@ -80,39 +85,17 @@ class HadoopFsDelegationTokenProvider extends HadoopDelegationTokenProvider with
 
 object HadoopFsDelegationTokenProvider extends Logging {
 
-  def disableFsCache(kyuubiConf: KyuubiConf, hadoopConf: Configuration): Configuration = {
-    // Avoid unnecessary disk io by not loading default resources
-    val newConf = KyuubiHadoopUtils.newHadoopConf(
-      kyuubiConf,
-      loadDefaults = false)
-
-    hadoopConf.iterator().asScala.foreach(e => newConf.set(e.getKey, e.getValue))
-
-    hadoopFSsToAccess(kyuubiConf, hadoopConf)
-      .foreach(fs => newConf.setBoolean(s"fs.${fs.getScheme}.impl.disable.cache", true))
-    newConf
-  }
-
-  def hadoopFSsToAccess(kyuubiConf: KyuubiConf, hadoopConf: Configuration): Set[FileSystem] = {
-    val filesystemsToAccess = kyuubiConf
-      .get(KyuubiConf.CREDENTIALS_HADOOP_FS_URIS)
-      .flatMap { uri =>
-        Try(new Path(uri).getFileSystem(hadoopConf)) match {
-          case Success(value) =>
-            Some(value)
-          case Failure(e) =>
-            warn(s"Failed to get Hadoop FileSystem instance by URI: $uri", e)
-            None
-        }
+  def validatedFsUris(kyuubiConf: KyuubiConf, hadoopConf: Configuration): Seq[URI] = {
+    val uris = kyuubiConf.get(KyuubiConf.CREDENTIALS_HADOOP_FS_URIS).map(new URI(_)) :+
+      FileSystem.getDefaultUri(hadoopConf)
```

Comment:
what happens if `FileSystem.getDefaultUri` throw exception ?

Reply:
@ulysses-you Updated code to hand exception caused by URI format